docs/content/manual/pre-release/node/backing-image-on-a-down-node.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Backing Image on a down node
+---
+
+1. Update the settings:
+   1. Disable Node Soft Anti-affinity.
+   2. Set Replica Replenishment Wait Interval to a relatively long value.
+2. Create a backing image with one of the following URLs:
+```
+   https://longhorn-backing-image.s3-us-west-1.amazonaws.com/parrot.qcow2
+   https://longhorn-backing-image.s3-us-west-1.amazonaws.com/parrot.raw
+```
+3. Create and 2 volumes with the backing image and attach them on different nodes.
```

Comment:
Do we need to check the related disk is added successfully with correct status?

Reply:
Yes. I will update this test